src/AzureRMTools.ts

```diff
@@ -109,7 +110,13 @@ export class AzureRMTools {
         const jsonOutline: JsonOutlineProvider = new JsonOutlineProvider(context);
         ext.jsonOutlineProvider = jsonOutline;
         context.subscriptions.push(vscode.window.registerTreeDataProvider("azurerm-vscode-tools.template-outline", jsonOutline));
-
+        context.subscriptions.push(
+            // tslint:disable-next-line:typedef
+            vscode.languages.registerCodeActionsProvider(armTemplateLanguageId, new RenameCodeActionProvider(async (document, position) => await this.getPositionContext(document, position, Cancellation.cantCancel)), {
```

Comment:
Use templateOrParameterDocumentSelector instead of armTemplateLanguageId - hopefully that should be enough to enough this in the param file, too (rename from there is new)